kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -694,6 +694,14 @@ object KyuubiConf {
       .checkValue(_ > 0, "the maximum must be positive integer.")
       .createWithDefault(10)
 
+  val SESSION_ENGINE_STARTUP_WAIT_COMPLETION: ConfigEntry[Boolean] =
+    buildConf("session.engine.startup.waitCompletion")
+      .doc("Whether to wait for completion after engine starts." +
```

Comment:
shall we make this config sounds more dangerous to adjust?

Reply:
How about this description?

> Whether to wait for completion after engine starts. If false, the startup process will be destroyed after the engine is started. Notice, it is only used in yarn-cluster mode, for yarn-client/local mode, the engine will be killed.

Reply:
How about
```
Note that only use it when the driver is not running locally, such as yarn-cluster mode; otherwise, the engine will be killed.
```

Reply:
> How about
> 
> > Note that only use it when the driver is not running locally, such as yarn-cluster mode; Otherwise, the engine will be killed.

+1, I wil add it